dnf5.spec

```diff
@@ -108,6 +108,13 @@ Provides:       dnf5-command(versionlock)
 %bcond_with    performance_tests
 %bcond_with    dnf5daemon_tests
 
+# Enable SOLVER_FLAG_FOCUS_NEW only for current rawhide
+%if 0%{?rhel} || 0%{?fedora} < 41
+%bcond_with    focus_new
+%else
+%bcond_without focus_new
+%endif
```

Comment:
Let's flip this, since we only care about old releases failing here.

```suggestion
# Disable SOLVER_FLAG_FOCUS_NEW only for old RHEL
%if 0%{?rhel} && 0%{?rhel} < 10
%bcond_with    focus_new
%else
%bcond_without focus_new
%endif
```

Reply:
We did wanted to enable this only for rawhide to test it out but since dnf5 is default only in rawhide maybe we could do this to simplify? @jan-kolarik

Reply:
Yes I think we should take the approach as with other features, to have upstream matching Rawhide and the older releases to use a patch turning this off. Just beware that for now we still want to keep this turned on just for Rawhide, so we also need to switch off for the upcoming Fedora 41 release.

Reply:
Why do we want to switch it off for Fedora at all? It's not used by default in older releases, and I'm not sure what's problematic about this flag. If you think there's potentially issues with it, we shouldn't do it at all. My understanding here is that the problem is related to having a new enough libsolv, but that's already in progress to land across all Fedora releases currently supported.

Reply:
> I'm not sure what's problematic about this flag

This is basically the reason. We are also not sure :slightly_smiling_face: So, we want to be more careful about it. We'll offer it first to Rawhide users and see if any issues emerge.

Reply:
Maybe we should think about making solver flags runtime-configurable like Zypper does...